src/mlpack/core.hpp

```diff
@@ -86,6 +86,7 @@
 #include <mlpack/core/dists/gaussian_distribution.hpp>
 #include <mlpack/core/dists/laplace_distribution.hpp>
 #include <mlpack/core/dists/gamma_distribution.hpp>
+#include <mlpack/core/data/cm.hpp>
```

Comment:
I would rename the file to something more descriptiv something like: ``confusion_matrix.hpp``.

Reply:
@zoq  , renamed file as confusion_matrix.hpp and confusion_matrix_impl.hpp.